### ISSUE TITLE
the size of the global map is 1

### ISSUE THREAD

**Issue by xy123001** [gaoxiang12/ORBSLAM2_with_pointcloud_map#20]
Hello,
I run it with kinect2 and all is right except that the global point only has a size of one
`generate point cloud for kf 103, size=78939
show global map, size=1
`
but when use the dataset, it is OK

**Comment by wzh-slam** [comment]
the same problem

**Comment by xy123001** [comment]
The resolution is missing in my .yaml  file .  When I add it, it works.

**Comment by wzh-slam** [comment]
could you be more  specific？

**Comment by wzh-slam** [comment]
may i ask a qq for communication？

**Comment by wzh-slam** [comment]
hi,xy123001.I am still trying fix it，could you show me your  .yaml file ？confused about the resolution you referring.

**Comment by xy123001** [comment]
Maybe  "PointCloudMapping.Resolution: 0.01 " in the last line of  "TUMX.yaml" is missing.  Please refer to Gaoxiang's TUM1.yaml

**Comment by wzh-slam** [comment]
yep,the problem have already been solved.still thank a lot!@xy123001
